Jenkinsfile

```diff
@@ -115,7 +115,7 @@ try {
         }
 
         recordIssues(
-          referenceJobName: "centreon-web/${env.REF_BRANCH}",
+          referenceJobName: "${env.PROJECT}/${env.REF_BRANCH}",
```

Comment:
@jeremyjaouen , it seems that "env.PROJECT" caused job's recording issues.
Why should it be reverted before : https://github.com/centreon/centreon/pull/10207